### ISSUE TITLE
Add feature to open source LimeSurvey (for EA Survey): JS to set display:block/display:none on later questions on a page, based on answers to earlier one

### ISSUE THREAD

**Issue by tog22** [tog22/eahub#2]
Obviously the JS is trivial and the LimeSurvey (PHP) interface is the only thing that'll take even a short time - though it should be easy. It's PHP. You could also hardcode this for the EA Survey to start with. Chance to contribute to nice open source project LimeSurvey.

**Comment by peterhurford** [comment]
Yeah, we should brainstorm this and then split it up into individual tasks.

**Comment by tog22** [comment]
I think the basic split is to hardcode an example question first, then add it to the LimeSurvey interface. I think that split's sufficient for someone to start work.

**Comment by peterhurford** [comment]
Oh, I posted that earlier comment on the wrong issue.  What I meant to say is that we should just use a survey program that does this automatically (e.g., Qualtrics or SurveyMonkey).  Thoughts?

**Comment by tog22** [comment]
Do you know of any? I'm not sure I should spend the time searching for and learning a new system. I had that survrymonkey walkthrough with chris, and it doesn't offer that feature (or anything nearly as sophisitcated as limesurvey does on that front). he thought limesurvey looked fine.

**Comment by tog22** [comment]
Because this tak doesn't require learning the rest of EA Hub and can be done without any real oversight, it'd actually be a good one to try some people on if you still want one :) People could install limesurvey themselves and try hacking its code directly.

**Comment by peterhurford** [comment]
Maybe I'm not understanding you right -- you're talking about hiding future questions based on past responses?

**Comment by tog22** [comment]
Yes, on the same page.

**Comment by peterhurford** [comment]
[That is easily doable in SurveyMonkey](http://help.surveymonkey.com/articles/en_US/kb/Question-Skip-Logic) ([and Qualtrics](http://www.qualtrics.com/university/researchsuite/basic-building/question-options/display-logic/)).

**Comment by peterhurford** [comment]
I think you can even [do it automatically in Lime Survey](https://manual.limesurvey.org/Setting_conditions#Hide.2FShow_Branching_Versus_Jump_Branching) though it doesn't look intuitive.

**Comment by tog22** [comment]
Yes, I used it plenty in LimeSurvey, but it only works for questions on
different pages, whereas this ticket is about doing it on the same page. It
sounds like SurveyMonkey has the same problem though I don't have a paid
account to test it with!

**Comment by peterhurford** [comment]
Yeah, that makes sense. I do think that support for same-page question hiding is pretty poor.

What's the downside of separating across multiple pages? Concerns of survey drop-off?

**Comment by tog22** [comment]
Yeah. Though don't have any evidence about thiS!

**Comment by peterhurford** [comment]
My best guess is it's fine to just have separate pages when needed.  Close this?

**Comment by tog22** [comment]
You could well be right, and I give that a bunch of weight. I think I'll just leave it, but keep the issue open and unassigned just in case anyone claims it. It could be good to have someone try their hand at LimeSurvey, and it's bound to be useful for us/LimeSurvey to have the ability to execute arbitrary JS on answer choice (the only real work here, as besides that it's just typing `$("#questionSelector").css("display","none;");`).
